climada/hazard/centroids/centr.py

```diff
@@ -1017,6 +1017,26 @@ def apply_point(df_exp):
                                  .map_partitions(apply_point, meta=Point)
                                  .compute(scheduler=scheduler))
 
+    @staticmethod
+    def _centroids_resolution(centroids):
+        """ Return resolution of the centroids in their units
```

Comment:
What is the motivation to make this method static and private?

Reply:
I had this function originally in the wildfire module and @chahank told me to move it to centroids. Happy to move it back.

Note for the wildfire module centroids need to be on a regular grid

Reply:
It does not really matter in which modules it is. At least the docstring message should be crystal clear.